zkevm-circuits/src/tx_circuit.rs

```diff
@@ -1941,6 +1946,38 @@ impl<F: Field> TxCircuitConfig<F> {
                 None,
                 Value::known(F::from(tx.tx_type as u64)),
             ),
+            (
+                AccessListAddressesLen,
+                None,
+                Value::known(F::from(access_list_address_size)),
+            ),
+            (
+                AccessListStorageKeysLen,
+                None,
+                Value::known(F::from(access_list_storage_key_size)),
+            ),
+            (
+                AccessListGasCost,
+                None,
+                Value::known(F::from(tx.access_list_gas_cost)),
+            ),
```

Comment:
This will not be needed

Reply:
Removed in commit [6880378](https://github.com/scroll-tech/zkevm-circuits/pull/1045/commits/68803783b1447b8fefe2c1fabd626beae51b8a83).